simon8491/src/main/java/workshop/UserAdd.java

```diff
@@ -0,0 +1,27 @@
+package workshop;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+@WebServlet("/user/add")
+public class UserAdd extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        getServletContext().getRequestDispatcher("/users/add.jsp").forward(req, resp);
+    }
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        User user = new User();
```

Comment:
CLEAN-CODE: pouzivat nemenne promenne vsude kde to je mozne v aplikaci.

```suggestion
final User user = new User();
```

**Duvod**

Používání neměnných proměnných final v Javě je dobré, protože zajišťuje, že hodnota proměnné se po jejím přiřazení nemůže změnit, což zvyšuje čitelnost a spolehlivost kódu. Pomáhá to také při optimalizaci kódu, protože kompilátor může lépe předvídat chování programu. Dále zajišťuje, že nebudou neúmyslné změny hodnoty proměnných, což snižuje riziko chyb, a také to usnadňuje práci s multithreadingem, kde je důležité, aby hodnoty nebyly modifikovány souběžně v různých vláknech.